DESCRIPTION

```diff
@@ -23,10 +23,15 @@ Description: Provides functionality for installing and updating
     github only packages. Other functionalities like checking conflicts
     are also provided. Inspired by tidyverse and dtupdate packages.
 License: GPL-3
-URL: https://github.com/natverse/natverse
+Encoding: UTF-8
+LazyData: true
+RoxygenNote: 7.0.2
+Language: en-GB
+URL: https://github.com/natverse/natverse, https://jefferis.github.io/nat/
 BugReports: https://github.com/natverse/natverse/issues
 Depends: 
     elmr,
+    neuprintr,
```

Comment:
Would you definitely also want to attach neuprintr (instead of loading them alone). please look at the options here: https://stackoverflow.com/questions/8637993/better-explanation-of-when-to-use-imports-depends    If you are sure about this, then go ahead!

Reply:
I would like people to be able to call library(natverse) and immediately be able to use neuPrintr functions because this might be a major natverse use as the hemibrain is out. So I might go for Depends. What do you think?

Reply:
Hmm, they would be able to use it if you put it under imports as well. However                        1) Do you have functions inside `neuprintr` that have same names as some other functions in `nat` or `elmr` maybe: In that case I would recommend `import`

Reply:
1. I think there are no name conflicts
2. While options could be set for connecting to neuPrint this is not done automatically. Rather the user is encouraged to use environment variables.

Reply:
Moved to imports for now